After git pull --rebase get the compile error

The I check the git Log ，my head is now at：

commit da2c9b8092259d00405034baad29742edbb115fc (HEAD -> main, origin/main, origin/HEAD)
Author: Romain Guy <romainguy@curious-creature.com>
Date:   Thu Feb 4 14:47:22 2021 -0800

    Fix doc typo


Execution failed for task ':filament-android:externalNativeBuildDebug'.
> Build command failed.
  Error while executing process D:\AppData\android-sdk\huawei-sdk\cmake\3.10.2.4988404\bin\ninja.exe with arguments {-C D:\oscar\github\filament\android\filament-android\.cxx\cmake\debug\arm64-v8a filament-jni}
  ninja: Entering directory `D:\oscar\github\filament\android\filament-android\.cxx\cmake\debug\arm64-v8a'
  [1/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/ColorGrading.cpp.o
  [2/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/Colors.cpp.o
  [3/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/EntityManager.cpp.o
  [4/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/Fence.cpp.o
  [5/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/Scene.cpp.o
  [6/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/MathUtils.cpp.o
  [7/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/IndirectLight.cpp.o
  [8/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/Camera.cpp.o
  [9/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/Engine.cpp.o
  [10/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/IndexBuffer.cpp.o
  [11/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/LightManager.cpp.o
  [12/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/MaterialInstance.cpp.o
  [13/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/SkyBox.cpp.o
  [14/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/nativewindow/Android.cpp.o
  [15/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/Material.cpp.o
  [16/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/Filament.cpp.o
  [17/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/RenderableManager.cpp.o
  [18/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/RenderTarget.cpp.o
  [19/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/Renderer.cpp.o
  [20/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/Stream.cpp.o
  [21/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/SwapChain.cpp.o
  [22/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/SurfaceOrientation.cpp.o
  [23/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/TextureSampler.cpp.o
  [24/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/TransformManager.cpp.o
  [25/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/Texture.cpp.o
  [26/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/View.cpp.o
  [27/30] Building CXX object CMakeFiles/filament-jni.dir/D_/oscar/github/filament/android/common/CallbackUtils.cpp.o
  [28/30] Building CXX object CMakeFiles/filament-jni.dir/src/main/cpp/VertexBuffer.cpp.o
  [29/30] Building CXX object CMakeFiles/filament-jni.dir/D_/oscar/github/filament/android/common/NioUtils.cpp.o
  [30/30] Linking CXX shared library libfilament-jni.so
  FAILED: libfilament-jni.so 
  cmd.exe /C "cd . && D:\AppData\android-sdk\huawei-sdk\ndk\22.0.7026061\toolchains\llvm\prebuilt\windows-x86_64\bin\clang++.exe --target=aarch64-none-linux-android21 --gcc-toolchain=D:/AppData/android-sdk/huawei-sdk/ndk/22.0.7026061/toolchains/llvm/prebuilt/windows-x86_64 --sysroot=D:/AppData/android-sdk/huawei-sdk/ndk/22.0.7026061/toolchains/llvm/prebuilt/windows-x86_64/sysroot -fPIC -g -DANDROID -fdata-sections -ffunction-sections -funwind-tables -fstack-protector-strong -no-canonical-prefixes -D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security  -std=c++17 -Wno-unused-command-line-argument -Wl,--hash-style=both -fno-stack-protector -fno-exceptions -fno-unwind-tables -fno-asynchronous-unwind-tables -fno-rtti -ffast-math -ffp-contract=fast -fvisibility-inlines-hidden -fvisibility=hidden -fomit-frame-pointer -ffunction-sections -fdata-sections -Wl,--gc-sections -Wl,-Bsymbolic-functions -O0 -fno-limit-debug-info  -Wl,--exclude-libs,libgcc.a -Wl,--exclude-libs,libgcc_real.a -Wl,--exclude-libs,libatomic.a -static-libstdc++ -Wl,--build-id=sha1 -Wl,--no-rosegment -Wl,--fatal-warnings -Wl,--no-undefined -Qunused-arguments -shared -Wl,-soname,libfilament-jni.so -o libfilament-jni.so CMakeFiles/filament-jni.dir/src/main/cpp/Camera.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/Colors.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/ColorGrading.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/Engine.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/EntityManager.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/Fence.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/IndexBuffer.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/IndirectLight.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/LightManager.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/Material.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/MaterialInstance.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/MathUtils.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/RenderableManager.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/Renderer.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/RenderTarget.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/Scene.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/SkyBox.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/Stream.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/SurfaceOrientation.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/SwapChain.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/Texture.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/TextureSampler.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/TransformManager.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/VertexBuffer.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/View.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/nativewindow/Android.cpp.o CMakeFiles/filament-jni.dir/src/main/cpp/Filament.cpp.o CMakeFiles/filament-jni.dir/D_/oscar/github/filament/android/common/CallbackUtils.cpp.o CMakeFiles/filament-jni.dir/D_/oscar/github/filament/android/common/NioUtils.cpp.o  D:/oscar/github/filament/out/android-release/filament/lib/arm64-v8a/libfilament.a D:/oscar/github/filament/out/android-release/filament/lib/arm64-v8a/libbackend.a D:/oscar/github/filament/out/android-release/filament/lib/arm64-v8a/libfilaflat.a D:/oscar/github/filament/out/android-release/filament/lib/arm64-v8a/libfilabridge.a D:/oscar/github/filament/out/android-release/filament/lib/arm64-v8a/libgeometry.a D:/oscar/github/filament/out/android-release/filament/lib/arm64-v8a/libibl-lite.a -llog -lGLESv3 -lEGL -landroid -ljnigraphics D:/oscar/github/filament/out/android-release/filament/lib/arm64-v8a/libutils.a D:/oscar/github/filament/out/android-release/filament/lib/arm64-v8a/libbluevk.a D:/oscar/github/filament/out/android-release/filament/lib/arm64-v8a/libsmol-v.a -latomic -lm && cd ."
  **ld: error: undefined symbol: VKSHADERS_PACKAGE**
  >>> referenced by VulkanBlitter.cpp
  >>>               VulkanBlitter.cpp.o:(filament::backend::VulkanBlitter::blitColor(VkCommandBuffer_T*, filament::backend::VulkanBlitter::BlitArgs)) in archive D:/oscar/github/filament/out/android-release/filament/lib/arm64-v8a/libbackend.a
  >>> referenced by VulkanBlitter.cpp
  >>>               VulkanBlitter.cpp.o:(filament::backend::VulkanBlitter::blitColor(VkCommandBuffer_T*, filament::backend::VulkanBlitter::BlitArgs)) in archive D:/oscar/github/filament/out/android-release/filament/lib/arm64-v8a/libbackend.a
  >>> referenced by VulkanBlitter.cpp
  >>>               VulkanBlitter.cpp.o:(filament::backend::VulkanBlitter::blitDepth(VkCommandBuffer_T*, filament::backend::VulkanBlitter::BlitArgs)) in archive D:/oscar/github/filament/out/android-release/filament/lib/arm64-v8a/libbackend.a
  >>> referenced 1 more times

another error:

cd out\cmake-release
cmake ^
    -G Ninja ^
    -DCMAKE_INSTALL_PREFIX=..\release\filament ^
    -DFILAMENT_ENABLE_JAVA=NO ^
    -DCMAKE_BUILD_TYPE=Release ^
    **-DFILAMENT_SUPPORTS_VULKAN=OFF ^**
    ..\..

ninja matc resgen cmgen

[116/445] Building CXX object third_party\spirv-tools\sour...keFiles\SPIRV-Tools-static.dir\val\validate_layout.cpp.obj
FAILED: third_party/spirv-tools/source/CMakeFiles/SPIRV-Tools-static.dir/val/validate_layout.cpp.obj
"D:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.25.28610\bin\Hostx64\x64\cl.exe"  /nologo /TP -DSPIRV_CHECK_CONTEXT -DSPIRV_COLOR_TERMINAL -DSPIRV_WINDOWS -I..\..\third_party\spirv-tools -I..\..\third_party\spirv-tools\include -Ithird_party\spirv-tools -I..\..\third_party\spirv-tools\external\SPIRV-Headers\include /DWIN32 /D_WINDOWS /W3 /GR /EHsc /std:c++latest /W0 /Zc:__cplusplus -D_USE_MATH_DEFINES=1 -D_CRT_SECURE_NO_WARNINGS -D_CRT_NONSTDC_NO_DEPRECATE /MD /O2 /Ob2 /DNDEBUG /MT /Zi   /MP4 -D_CRT_SECURE_NO_WARNINGS -D_SCL_SECURE_NO_WARNINGS /wd4800 /WX /EHs /showIncludes /Fothird_party\spirv-tools\source\CMakeFiles\SPIRV-Tools-static.dir\val\validate_layout.cpp.obj /Fdthird_party\spirv-tools\source\CMakeFiles\SPIRV-Tools-static.dir\SPIRV-Tools-static.pdb /FS -c ..\..\third_party\spirv-tools\source\val\validate_layout.cpp
cl: 命令行 warning D9025 :正在重写“/W3”(用“/W0”)
cl: 命令行 warning D9025 :正在重写“/MD”(用“/MT”)
..\..\third_party\spirv-tools\source\val\validate_layout.cpp(20): fatal error C1083: 无法打开包括文件: “OpenCLDebugInfo100.h”: No such file or directory

--- a/android/filament-android/CMakeLists.txt
+++ b/android/filament-android/CMakeLists.txt
@@ -37,6 +37,10 @@ add_library(bluevk STATIC IMPORTED)
 set_target_properties(bluevk PROPERTIES IMPORTED_LOCATION
         ${FILAMENT_DIR}/lib/${ANDROID_ABI}/libbluevk.a)
 
+add_library(vkshaders STATIC IMPORTED)
+set_target_properties(vkshaders PROPERTIES IMPORTED_LOCATION
+        ${FILAMENT_DIR}/lib/${ANDROID_ABI}/libvkshaders.a)
+
 add_library(smol-v STATIC IMPORTED)
 set_target_properties(smol-v PROPERTIES IMPORTED_LOCATION
         ${FILAMENT_DIR}/lib/${ANDROID_ABI}/libsmol-v.a)
@@ -101,6 +105,7 @@ target_link_libraries(filament-jni
     PRIVATE utils
     $<$<STREQUAL:${FILAMENT_ENABLE_MATDBG},ON>:matdbg>
     $<$<STREQUAL:${FILAMENT_SUPPORTS_VULKAN},ON>:bluevk>
+    $<$<STREQUAL:${FILAMENT_SUPPORTS_VULKAN},ON>:vkshaders>
     $<$<STREQUAL:${FILAMENT_SUPPORTS_VULKAN},ON>:smol-v>
 )
 

--- a/filament/backend/CMakeLists.txt
+++ b/filament/backend/CMakeLists.txt
@@ -326,6 +326,7 @@ target_link_libraries(${TARGET} PRIVATE
 # ==================================================================================================
 set(INSTALL_TYPE ARCHIVE)
 install(TARGETS ${TARGET} ${INSTALL_TYPE} DESTINATION lib/${DIST_DIR})
+install(TARGETS vkshaders ${INSTALL_TYPE} DESTINATION lib/${DIST_DIR})
 install(DIRECTORY ${PUBLIC_HDR_DIR}/backend DESTINATION include)
 
 # ==================================================================================================

